package.json

```diff
@@ -21,12 +21,12 @@
   },
   "publishConfig": {
     "registry": "https://registry.npmjs.org/",
-    "always-auth": true
+    "access": "public"
   },
```

Comment:
Can you consolidate this in the `.npmrc` file?

I didn't find anything on the [standards-and-best-practices](https://github.com/goodeggs/standards-and-best-practices) repo related to prefer dotfile/configuration files to package.json configs, so, the `.npmrc` should be fine? 😅

Reply:
I'm following this guide to the letter: https://goodeggs.atlassian.net/wiki/spaces/ENG/pages/459833453/Using+NPM+and+Yarn#How-to-configure-a-public-module.

Keep in mind `.npmrc` applies to all uses of NPM, whereas `package.json#publishConfig` is config that overrides `.npmrc` (etc.) config specifically when in the context of publishing a package. `access` isn't meaningful in any other context, so we define it in the publish context.

Reply:
Got it! Cool!